webrtc/RTCConfiguration-iceCandidatePoolSize.html

```diff
@@ -88,8 +96,9 @@
 */
 
 test(() => {
+  const pc = new RTCPeerConnection({});
+  assert_idl_attribute(pc, "setConfiguration");
   assert_equals(typeof pc.setConfiguration, "function", "RTCPeerConnection.prototype.setConfiguration is not implemented");
```

Comment:
This become redundant with the `assert_idl_attribute` above. We probably don't want to assert everything too defensively.